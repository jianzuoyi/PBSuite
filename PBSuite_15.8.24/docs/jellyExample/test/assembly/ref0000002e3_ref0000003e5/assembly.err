2017-10-26 16:40:21,868 [INFO] Running /its1/GB_BT2/jianzuoyi/pipeline/biosoft/01.Assembly/PBSuite/PBSuite_15.8.24/bin/Assembly.py /its1/GB_BT2/jianzuoyi/pipeline/biosoft/01.Assembly/PBSuite/PBSuite_15.8.24/docs/jellyExample/test/assembly/ref0000002e3_ref0000003e5 --nproc=4
2017-10-26 16:40:22,608 [INFO] Preunited 0 reads
2017-10-26 16:40:22,725 [INFO] Running /its1/GB_BT2/jianzuoyi/pipeline/biosoft/01.Assembly/PBSuite/PBSuite_15.8.24/bin/Assembly.py /its1/GB_BT2/jianzuoyi/pipeline/biosoft/01.Assembly/PBSuite/PBSuite_15.8.24/docs/jellyExample/test/assembly/ref0000002e3_ref0000003e5 --nproc=4
2017-10-26 16:40:22,727 [INFO] Extracting tails
2017-10-26 16:40:22,727 [INFO] Parsed 19 reads
2017-10-26 16:40:22,728 [INFO] Found 18 tails
2017-10-26 16:40:22,728 [INFO] 1 reads had double tails
2017-10-26 16:40:22,728 [INFO] Mapping Tails
2017-10-26 16:40:22,754 [INFO] [0, '[INFO] 2017-10-26T16:40:22 [blasr] started.\n[INFO] 2017-10-26T16:40:22 [blasr] ended.\n', None]
2017-10-26 16:40:22,754 [INFO] Consolidating alignments
2017-10-26 16:40:22,756 [INFO] 17 tails mapped
2017-10-26 16:40:22,761 [INFO] 15 reads span
2017-10-26 16:40:22,761 [INFO] 17 reads extend flank 1
2017-10-26 16:40:22,761 [INFO] 16 reads extend flank 2
2017-10-26 16:40:22,762 [INFO] estimated fill len 192
2017-10-26 16:40:22,762 [INFO] estimated flank 1 extend len 701
2017-10-26 16:40:22,763 [INFO] estimated flank 2 extend len 423
2017-10-26 16:40:22,807 [INFO] Cleaning 10 temp files
2017-10-26 16:40:22,807 [INFO] Finished
