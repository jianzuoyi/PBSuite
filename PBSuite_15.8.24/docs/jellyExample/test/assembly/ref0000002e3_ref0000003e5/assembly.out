['/its1/GB_BT2/jianzuoyi/pipeline/biosoft/01.Assembly/PBSuite/PBSuite_15.8.24/docs/jellyExample/test/assembly/ref0000002e3_ref0000003e5/primary_XLAjHP.m4', '/its1/GB_BT2/jianzuoyi/pipeline/biosoft/01.Assembly/PBSuite/PBSuite_15.8.24/docs/jellyExample/test/assembly/ref0000002e3_ref0000003e5/query_5lCKd6.fastq', '/its1/GB_BT2/jianzuoyi/pipeline/biosoft/01.Assembly/PBSuite/PBSuite_15.8.24/docs/jellyExample/test/assembly/ref0000002e3_ref0000003e5/target_M3GALW.fasta', '-t', '200', '-n', '4', '-o', '/its1/GB_BT2/jianzuoyi/pipeline/biosoft/01.Assembly/PBSuite/PBSuite_15.8.24/docs/jellyExample/test/assembly/ref0000002e3_ref0000003e5/onFlank_t9WRTY.m5']
